packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -79,15 +103,24 @@ export default class FlyoutHelpStore {
 
         const getNextBlock = async (xml, current_index, direction) => {
```

Comment:
`direction` is a boolean and should be named as such, e.g. `is_next` or `is_previous` is more appropriate.

Reply:
done

Reply:
It's still `direction` :P